### ISSUE TITLE
Question: When we can expect next release?

### ISSUE THREAD

**Issue by armano2** [eslint/typescript-eslint-parser#575]
I have small question about next release, when we can expect to have it live?

in [eslint-plugin-typescript](https://github.com/bradzacher/eslint-plugin-typescript) we are block due to issues with traversing AST

- https://github.com/bradzacher/eslint-plugin-typescript/pull/174
- https://github.com/bradzacher/eslint-plugin-typescript/pull/181
- https://github.com/bradzacher/eslint-plugin-typescript/pull/117

**Comment by corbinu** [comment]
I was wondering if it would be good to setup nightly builds? Or at least an easier way to release RCs. I am going to suggest this in typescript-estree.

This would be really for use with the nightly typescript version and only for those that want to be on the bleeding edge. It would also be good for situations like this were devs want to test out a bunch of changes in the matching plugins.

I am happy to help if this sounds like a good idea.

**Comment by armano2** [comment]
ohh, i was asking more about releasing fixes from master, rather than supporting RC versions of TS :)

**Comment by corbinu** [comment]
@armano2 I know sorry maybe I should have just opened a separate issue just was thinking it would make development easier.

**Comment by platinumazure** [comment]
We're still trying to come up with the best ownership plan and release schedule (or at least release practices) after JamesHenry left the project. I greatly appreciate your patience as we continue to figure it out.

I should have time tonight to take a look and see what might still be mergeable, and can cut a release if everything is looking good.

**Comment by kaicataldo** [comment]
@corbinu My biggest concern with nightly releases is the maintenance burden both in typescript-estree and this project. Any AST changes would need to be account for (potentially in both projects) to ensure compatibility with ESLint. I'm not sure the overhead is worth it, given that the ESLint team is already maintaining so many packages.

**Comment by corbinu** [comment]
@kaicataldo I was thinking something automated so no real burden increase. If there is breakage people can just pin to a version. This is how most people use the typescript@nightly releases. 

I am just trying to find some solution to the status quo which is A) The new version is never ready when a new release of TypeScript drops B) I know I am not the only one who ends up maintaining forks to have access to the latest changes

Basically I am just suggesting we make what people are already doing official and setup a script that releases off of master nightly (and includes whatever is in typescript-estree in master or in nightly if James agrees to this idea for that project also) so people don't have to handle in via custom forks.

**Comment by armano2** [comment]
@kaicataldo i started adding correct typescript types in estree project, its going to not compile if there are going to misstatements with eslint / typescript

**Comment by platinumazure** [comment]
Apologies for not having time to release over the weekend.

I've merged everything semver-patch except for 2 changes:

* #573 (due to conflicts)
* #558 (because we're still trying to agree on an approach)

I'd like to do a patch release with what's currently in master now, and then we can review and potentially merge the semver-minor #568.

@kaicataldo @mysticatea If you have any objections, please state them here or in our team chat; otherwise, I'll do the release later today (US time).

**Comment by corbinu** [comment]
@platinumazure We are also due for a major now that Typescript 3.2 was released. I can open a PR really quick bumping typescript-estree to 6.0

**Comment by platinumazure** [comment]
@corbinu Sounds good but I think I want to cut this patch release without bumping typescript-estree to a new major version. So no rush on that PR :smile:

To be clear, I'm anticipating cutting 2 releases in the next couple of days: Patch release today (hopefully), and a minor or major release with some of the new stuff that's coming in soon. I'd like the typescript-estree bump in the second release.

**Comment by platinumazure** [comment]
@armano2 That's my goal:

> To be clear, I'm anticipating cutting 2 releases in the next couple of days: Patch release today (hopefully), and a minor or major release with some of the new stuff that's coming in soon. I'd like the typescript-estree bump in the second release.

**Comment by platinumazure** [comment]
I've just released [`typescript-eslint-parser@21.0.2`](https://github.com/eslint/typescript-eslint-parser/releases/tag/v21.0.2).

Let's see if we can land the typescript-estree upgrade and the parser services PR this week, then do a major release (major only by convention since the supported Typescript range is changing).

**Comment by armano2** [comment]
thank you

**Comment by platinumazure** [comment]
I'm going to close this as we've done a patch release and outlined the rough plan for the next release. Thanks!
